metrics_utility/automation_controller_billing/helpers.py

```diff
@@ -9,7 +9,10 @@
 from dateutil import parser
 from dateutil.relativedelta import relativedelta
 
-from metrics_utility.exceptions import UnparsableParameter
+from metrics_utility.exceptions import DateFormatError, UnparsableParameter
+
+
+DAY_OR_MONTH_REGEX_PATTERN = r'.^\d+(d|day|days|m|mo|month|months)$'
```

Comment:
```suggestion
DAY_OR_MONTH_REGEX_PATTERN = r'^\d+(d|day|days|m|mo|month|months)$'
```

otherwise this requires a character before the start of the line